gulp/tasks/handlebars.js

```diff
@@ -64,7 +64,7 @@ gulp.task( 'handlebars:index',
     charts: charts
   }
 
-  console.log(chartData.charts)
+  // console.log(chartData.charts)
```

Comment:
Do we need this line anymore? If not, can we just remove it rather than commenting it out?